L4/API-skryptowe-zadanie1+2+3.py

```diff
@@ -0,0 +1,81 @@
+import requests
+import json
+import datetime
+import numpy as np
+import matplotlib.pyplot as plt
+
+def smallerPieceOfData (currency,lastDays):
+    
+    today=datetime.datetime.now()  
+    todayDate=today.strftime("%Y-%m-%d")
+    chosen=today-datetime.timedelta(days=(lastDays+1))
+    chosenDate=chosen.strftime("%Y-%m-%d")
+    resp=requests.get("http://api.nbp.pl/api/exchangerates/rates/a/{currency}/{chosenDate}/{todayDate}/?format=json".format(currency=currency,chosenDate=chosenDate, todayDate=todayDate))
+    if(resp.status_code!=200):
+        raise Exception('Error {} url nbp api'.format(resp.status_code))
+    else:
+        print('{}'.format(resp.json()))
+    return resp
+
+def exchangeRates (currency, days):
+    resp=[]
+    if (days>185):
```

Comment:
takie rzeczy jak to 185 powinno być w STALEJ. Poza tym kod w takiej postaci jest dość mało uniwersalny i ciężko byłoby dostosować tę funkcję do pracy z każdym przedziałem dat.